packages/plugin-slider/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "plugin-slider",
+  "version": "0.0.1",
+  "description": "Will provide a slider functionality in order to begin experiments      ",
+  "type": "module",
+  "main": "dist/index.cjs",
+  "exports": {
+    "import": "./dist/index.js",
+    "require": "./dist/index.cjs"
+  },
+  "typings": "dist/index.d.ts",
+  "unpkg": "dist/index.browser.min.js",
+  "files": [
+    "src",
+    "dist"
+  ],
+  "source": "src/index.ts",
+  "scripts": {
+    "test": "jest",
+    "test:watch": "npm test -- --watch",
+    "tsc": "tsc",
+    "build": "rollup --config",
+    "build:watch": "npm run build -- --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "",
+    "directory": "./"
+  },
+  "keywords": [
+    "jsPsych"
+  ],
+  "author": {
+    "name": "Vishnu Lakshman",
+    "url": "https://github.com/lakshmanvishnu"
+  },
+  "license": "MIT",
+  "bugs": {
+    "url": "/issues"
+  },
+  "homepage": "/plugin-slider/README.md",
+  "dependencies": {
+    "@citation-js/core": "^0.7.14",
```

Comment:
I don't think the @citation-js dependencies should be in here. did those get added?